electrode-ota-server-model-acquisition/src/acquisition.js

```diff
@@ -72,7 +73,7 @@ export default (options, dao, weighted, _download, manifest, logger) => {
                         packageHash: pkg.packageHash,
                         description: pkg.description,
                         // true == there is an update but it requires a newer binary version.
-                        updateAppVersion: version.lt(paramAppVersion, pkgAppVersion),
+                        updateAppVersion: isTargetBinaryVersion,
```

Comment:
`updateAppVersion` means the mobile client has a binary client version that is lower and needs to be updated to newer binary version.  
how is that the same as `isTargetBinaryVersion`?

Reply:
using `ltr(version, range)` that returns true if version is less than all the versions possible in the range.